tsconfig.json

```diff
@@ -2,6 +2,7 @@
     "compilerOptions": {
         "strict": true,
         "target": "ES2024",
+        "module": "ES2022",
```

Comment:
Module and moduleResolution are being taken care of in another PR?

Reply:
Why ES022 now?

Reply:
Because ES2022 is what we agreed on for module... I thought.

Yes module and module resolution are in a separate [issue](https://github.com/demergent-labs/azle/issues/2661). Though es2022 is the one I am planning on using for that pr.